WordPress/gutenberg
# Toolbar: move all subcomponents under the same folder

## What?


Unify under the same parent `toolbar` folder all related subcomponents.

## Why?


This PR is in preparation to two refactors:

- unifying Storybook examples while moving away from knobs
- rewriting the components in TypeScript

Moving all of the subcomponents under the same folder will reduce overhead during those two refactors, while keeping all these changes in a separate PR will also greatly reduce diff noise.

## How?


- moved each `Toolbar*` subcomponent in its own folder, under the `toolbar` folder
- moved all Storybook examples under the `stories` folder
- moves all tests under the `tests` folder
- updated import & export paths across the project
- removed out-of-date paths from `tsconfig.json`

## Testing Instructions





- Make sure the project builds correctly and all tests pass (both web and native versions)
- Make sure Storybook examples work exactly like on `trunk`
